src/modules/utils/panel/screens/3dprinter/WatchScreen.cpp

```diff
@@ -331,6 +331,7 @@ void WatchScreen::redraw()
 void WatchScreen::draw_graphics()
 {
     THEPANEL->lcd->clear();
+    THEPANEL->lcd->setBackground(false);
```

Comment:
Won't this then stay false, so this will not fix the problem? It needs to reset it so that other writes use the old functionality?

Reply:
It is set back to true whenever clear() is called. I can set it explicitly if that would make things more clear.

Reply:
That is pretty convoluted ,it is already getting a little too messy for my tastes anyway. So yes it needs to be explicit.

Reply:
Sure, let me know if there is a cleaner approach to anything.